Scripts/configure-hooks

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+HOOKS=$(git rev-parse --git-path hooks)
+ROOT=$(git rev-parse --show-toplevel)
+
+# Symlink the hooks from scripts/hooks to the $HOOKS directory
+for hook in "${ROOT}"/scripts/hooks/*; do
```

Comment:
**nitpick** The folder is `Scripts`, but we're using the lowercase version her

Reply:
Ah good catch! 👀